load.php

```diff
@@ -49,7 +53,9 @@ function bootstrap( $wp_debug_enabled ) {
 		Admin\bootstrap();
 	}
 
-	require_once __DIR__ . '/lib/ses-to-cloudwatch/plugin.php';
+	if ( HM_ENV_TYPE !== 'local' ) {
+		require_once __DIR__ . '/lib/ses-to-cloudwatch/plugin.php';
```

Comment:
I'm thinking we might want a different way to handle these, rather than hardcoding allowing it. Perhaps just defaulting the a config value?

Reply:
This was a quick option, given it's code designed to run on AWS I'm not sure what a neater way might look like unless we consolidate how everything in `lib` loaded based on the values in `$hm_platform`